### DISCUSSION TITLE
Windows Authentication for the SQL server is not working

### DISCUSSION THREAD

**Discussion by himabindutellakula** [schemaspy/schemaspy#1471] [Q&A]
I tried the below command -
java -Djava.library.path=Windows_Auth_DLL/mssql-jdbc_auth-12.6.2.x64.dll -jar schemaspy-6.2.4.jar -configFile config.file -sso -connprops integratedSecurity\=true;trustServerCertificate\=true

And I am getting the error
**This driver is not configured for integrated authentication.**

**Comment by npetzall** [comment]
https://schemaspy.readthedocs.io/en/latest/databases/index.html#mssql

There is both a caveat regarding `-connprops` which has to do with which console/terminal that you're using you might need to use either `\=` or `\\=` between key and value.

LibraryPath is a bit of a hassel, shouldn't be a file but a folder and I'm a bit unsure about the actually syntax.

https://github.com/schemaspy/schemaspy/issues/566#issuecomment-502406717

Has both the connprops and an example of library.path.

**Reply by npetzall** [reply]
> This driver is not configured for integrated authentication.

Stackoverflow mentions this as a output when it's unable to load the dll file.
